docker/nix-build.sh

```diff
@@ -4,6 +4,13 @@ set -euo pipefail
 SPAWN_SHELL=${SPAWN_SHELL:-false}
```

Comment:
Can we please change the way docker displays logs from `nix-build.sh` ? So it does not display logs in an interactive way. There's a limit in docker of 1 MB now which should be solved by that as well.

Reply:
Procedure described by`BUILD.md` should force docker disable log clipping (it's ugly...).